module-apps/application-settings/windows/bluetooth/BluetoothCheckPasskeyWindow.hpp

```diff
@@ -22,6 +22,6 @@ namespace gui
         Image *image = nullptr;
         Label *label = nullptr;
         Text *text   = nullptr;
-        std::unique_ptr<BluetoothSettingsModel> bluetoothSettingsModel;
+        std::shared_ptr<BluetoothSettingsModel> bluetoothSettingsModel;
```

Comment:
:+1:

Reply:
this was a funny one - it was defined in the application and here, under the same name and I spend quite a bit of time wondering - why the set device doesn't match the device received in application :)

Reply:
tbh - it's already shared in ApplicationCommon